examples/plot_incr_kmeans.py

```diff
@@ -0,0 +1,274 @@
+"""
```

Comment:
Perhaps it would be easier for the reader to start with the Active Learning experiment (after all this is the purpose of the library), and then as a deep dive into the IncrementalKMeans approach have the rest of the explanations ?

Reply:
Yes, I was hesitating to do it in this sense. I will swap them!

Reply:
Fixed in upcoming commit.